examples/src/main/scala/org/apache/spark/examples/streaming/CustomReceiver.scala

```diff
@@ -82,9 +82,9 @@ class CustomReceiver(host: String, port: Int)
    var socket: Socket = null
    var userInput: String = null
    try {
-     logInfo("Connecting to " + host + ":" + port)
+     logInfo(s"Connecting to $host $port")
```

Comment:
Nit: we could make the string consistent with the one two lines below by adding a colon

Reply:
Done, addressed